src/lib/y2security/security_policies/manager.rb

```diff
@@ -140,6 +146,35 @@ def env_policies
 
         ENV[key].split(",")
       end
+
+      SERVICE_NAME = "ssg-apply".freeze
+      private_constant :SERVICE_NAME
+
+      # Adds the package and enables the service to remedy the system after the installation
+      def enable_service
+        return if enabled_services.include?(SERVICE_NAME)
+
+        enabled_services << SERVICE_NAME
+        Yast::PackagesProposal.AddResolvables("security", :package, [SERVICE_NAME])
```

Comment:
Where is the package coming from? Last I checked, it was in a special add-on product

Reply:
At this point, I do not know. I would expect the software solver to fail if the package is not available (I added the package in a DUD).